### PR TITLE
Closes #3352: Update CONTRIBUTING.md file with information about issues and PRs.

### LINKED ISSUE #3352: Update readme file with info about issues, PRs, milestones, and projects.
Update README.md to include details about how to use issues, PRs, milestones, and projects.

### PR DESCRIPTION
**Pull request by danahertzberg**
## Description
Update README.md file with information about issues and PRs.

## Related issues
Closes #3352

## How to test
Review documentation change to ensure it is understandable.

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [x] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by bberndt-uaz — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ how to contribute including:
 * How to [submit a pull request](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#pull-requests)
 * [Local development](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#local-development)
 
+### Creating Issues and Pull Requests
+
+Create an issue in the associated repository to a request a change. That issue can then be added to a milestone to plan work for a future release.
+
+Create a pull request (PR) to change the code within a specific repository. PRs often have an issue associated with them, but not always (like in the case of dependabot making PRs). When possible, always link an issue to a PR.
```
Comment (trackleft) [inline README.md:38]:
```suggestion
[Create a pull request (PR)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to change the code within a specific repository. PRs often have an issue associated with them, but not always (like in the case of dependabot making PRs). When possible, always link an issue to a PR. 
```

File: README.md [hunk 2]
```diff
@@ -31,6 +31,14 @@ how to contribute including:
 * How to [submit a pull request](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#pull-requests)
 * [Local development](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#local-development)
 
+### Creating Issues and Pull Requests
+
+Create an issue in the associated repository to a request a change. That issue can then be added to a milestone to plan work for a future release.
+
+Create a pull request (PR) to change the code within a specific repository. PRs often have an issue associated with them, but not always (like in the case of dependabot making PRs). When possible, always link an issue to a PR. 
+
+Projects are created in conjunction with specific releases -- patch, minor, or major. All PRs must be added to at least one project to indicate the associated Quickstart release. These projects will aid in advance planning of release testing, writing release notes, etc.
```
Comment (trackleft) [inline README.md:40]:
```suggestion
[Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) are created in conjunction with specific releases -- patch, minor, or major. All PRs must be added to at least one project to indicate the associated Quickstart release. These projects will aid in advance planning of release testing, writing release notes, etc.
```

Comment (trackleft) [reply README.md:40]: Alternate "releases" link https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases

Comment (trackleft) [reply README.md:40]: While the language in this PR can be applied to all the repositories under the az-digital namepace(and might be best added to the namespace readme or wiki) for this PR specifically I think we can link to the project/repository specific release policy.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -54,7 +59,7 @@ We follow Drupal
 
 ## Pull requests
```
Comment (danahertzberg) [inline CONTRIBUTING.md:60]:
```suggestion
## Pull request guidelines
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -28,6 +28,12 @@ team of web-focused volunteers that meet weekly to create projects like [Arizona
 
 ## Submitting a bug/issue/feature request
```
Comment (danahertzberg) [inline CONTRIBUTING.md:29]:
```suggestion
## Creating Issues and Pull Requests
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -42,19 +48,18 @@ az-digital-security@list.arizona.edu
 We use [GitHub Issues](https://github.com/az-digital/az_quickstart/issues) to
 keep track of issues and bugs.
 
-If you don't see the specific issue or bug after looking at the
-[AZ Quickstart Project](https://github.com/orgs/az-digital/projects/1), please
-create a new issue with proper description of bug or details related to new
-feature request.
+If you don't see the specific issue or bug in the [az_quickstart issue list](https://github.com/az-digital/az_quickstart/issues) 
+or [arizona_bootstrap issue list](https://github.com/az-digital/arizona-bootstrap/issues), please create a new issue with proper description 
+of bug or details related to new feature request.
```
Comment (bberndt-uaz) [inline CONTRIBUTING.md:53]:
```suggestion
or [arizona_bootstrap issue list](https://github.com/az-digital/arizona-bootstrap/issues), please create a new issue with as much detail as possible about the bug or feature request.
```

**Comment by bberndt-uaz** [comment]
The heading on line 46 could be updated as well, from "General bug/new feature request" to something like "General Bug Fix and Enhancement Issues" or "Bug Reports and Feature Requests".

**Review by bberndt-uaz — DISMISSED** [review]

**Review by tadean — DISMISSED** [review]

**Comment by danahertzberg** [comment]
- [ ] Update README link
- [ ] Add info about milestones not being in other repos (any PR can be in a project)
   - Issues created in the primary repo, should also be added to the milestone

**Review by trackleft — APPROVED** [review]

**Review by bberndt-uaz — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Pull request merged**